File: packages/java/src/main/java/com/logolabs/inkvec/InvalidImageException.java
package com.logolabs.inkvec;

/** The input is not a decodable image, or raw pixels do not match the size given. */
public final class InvalidImageException extends InkvecException {
    private static final long serialVersionUID = 1L;

    InvalidImageException(String message) {
        super(message);
    }
}

File: packages/php/src/InvalidImageException.php
<?php

declare(strict_types=1);

namespace LogoLabs\Inkvec;

/**
 * The bytes are not an image Inkvec can decode, or the raw pixels do not match the width
 * and height given (`INKVEC_ERR_INVALID_IMAGE`).
 */
final class InvalidImageException extends InkvecException
{
}
